deploy/crds/nmstate.io_nmstates_crd.yaml

```diff
@@ -5,14 +5,14 @@ metadata:
 spec:
   group: nmstate.io
   names:
-    kind: NMstate
```

Comment:
You need to rebase the PR seems like.

Reply:
Aha, https://github.com/nmstate/kubernetes-nmstate/pull/521 is not merged yet. Sorry